llvm/include/llvm/Analysis/AltMathLibFuncs.def

```diff
@@ -1168,6 +1169,71 @@ TLI_DEFINE_ALTMATHFUNC(Intrinsic::fpbuiltin_sqrt, Type::FloatTyID, FIXED(8), "__
 TLI_DEFINE_ALTMATHFUNC(Intrinsic::fpbuiltin_sqrt, Type::FloatTyID, FIXED(8), "__svml_sqrtf8_ha", 0.6)
 #endif
 #undef TLI_DEFINE_SVML_ALTMATHFUNCS
+#if defined(TLI_DEFINE_NO_ALTMATHFUNCS)
+
+// NO_ALTMATH_LIB Single precision implementations
+TLI_DEFINE_ALTMATHFUNC(Intrinsic::fpbuiltin_sin, Type::FloatTyID, FIXED(1), "sin", 0.6)
```

Comment:
The accuracy listed here doesn't correspond to what's provided by the GNU math library. I don't think we should commit this.

Reply:
Thanks for pointing this out!